### PR TITLE
RK-717 - Python sanic example 

### PR DESCRIPTION
**Pull request by ManuelNemirovsky**

### CODE REVIEW

File: .jenkins/Jenkinsfile [hunk 1]
```diff
@@ -11,7 +11,7 @@ podCreator.initMain(label: label) {
             def myRepo = checkout scm
             env.GIT_BRANCH = myRepo.GIT_BRANCH.replaceAll('/|_', '-')
             // Replacing characters we cannot support in deploy branch
-            def testFolder = "master".equals(env.GIT_BRANCH) ? "regression-test" : "sandbox-regression-test"
+            def testFolder = "python-sanic".equals(env.GIT_BRANCH) ? "regression-test" : "sandbox-regression-test"
```
Comment (itielshwartz) [inline .jenkins/Jenkinsfile]: remove

Comment (ManuelNemirovsky) [reply .jenkins/Jenkinsfile]: removed "python-sanic"

File: python-sanic/sanic_rookout.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+async def test_await(request):
+    import asyncio
+    await asyncio.sleep(5)
+    return response.text("I'm feeling sleepy")
+
+
+
+async def load_rookout():
+    try:
+        if os.environ["ROOKOUT_TOKEN"]:
```
Comment (itielshwartz) [inline python-sanic/sanic_rookout.py]: why do they need it?

File: python-sanic/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+A sample application for debugging Python Sanic apps using Rookout.
+
+Before following this guide we recommend reading the basic [Python + Rookout](https://docs.rookout.com/docs/sdk-setup.html) guide.
+
+This sample may be out of date. If you face any issues, please reach out to mailto:support@rookout.com and let us know.
+
+## Running Sanic Server with Rookout
+
+1. *Clone and install dependencies*:
```
Comment (itielshwartz) [inline python-sanic/README.md]: write about the diff between with worker

File: python-sanic/sanic_rookout.py (hunk 2 above)
Comment (ManuelNemirovsky) [reply python-sanic/sanic_rookout.py]: The workers are handling requests in parallel, so if you will import rook at the beginning of the code, rook will shutdown with this error -  `[Rookout] Rookout does not support running in forking processes. Please import only after forking.`
A good way to handle this error is loading rook to each worker only after forking is done.

File: python-sanic/README.md (hunk 3 above)
Comment (ManuelNemirovsky) [reply python-sanic/README.md]: I wrote an explanation about the workers in the `Rookout Integration Explained` section 
If its not enough please tell me :)

File: python-sanic/sanic_rookout.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+def start_server():
+    app.add_task(load_rookout())
+    app.run(host='0.0.0.0',
+            port=os.environ.get('PORT', 5000),
+            access_log=False,
+            debug=False,
+            workers=3)
+
+start_server()
+load_rookout()
```
Comment (itielshwartz) [inline python-sanic/sanic_rookout.py]: why do i need the load rookout here also?

Comment (ManuelNemirovsky) [reply python-sanic/sanic_rookout.py]: I did that for checking something and forgot to delete

**Pull request merged**